regparser/commands/outline_depths.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from regparser.tree.depth.derive import derive_depths
+
+import click
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument('markers', type=click.STRING, required=True)
```

Comment:
Thanks - thinking about it, it may be easier to use as one string because the calling app needs to pass just one command line arg.